Source/WTF/wtf/text/StringView.h

```diff
@@ -68,13 +68,12 @@ class StringView final {
     StringView(const String&);
     StringView(const StringImpl&);
     StringView(const StringImpl*);
-    StringView(const LChar*, unsigned length);
-    StringView(const UChar*, unsigned length);
-    StringView(const char*, unsigned length);
+    StringView(std::span<const LChar>);
+    StringView(std::span<const UChar>);
+    StringView(std::span<const char>);
```

Comment:
Might want to remove this eventually since encoding of non-ASCII `char` is ambiguous, unlike `LChar` and `char8_t`.